src/test/java/chambre/morsecode/WordleGameTest.java

```diff
@@ -0,0 +1,19 @@
+package chambre.morsecode;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WordleGameTest {
```

Comment:
Can you also write a test for `guess()` returning the an array that's incorrect?

Reply:
I used the word frindle that is not in the dictionary- but if the word that is generated contains any of the letters from frindle then wont it return wrong place and my test is useless.

Reply:
There's still a way to structure your test so that it won't have that issue.

Reply:
Got it.

Reply:
You are still missing a test for guess which returns incorrect results.